app/components/lazy_project_tag_facet_component.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="card facet-limit blacklight-exploded_project_tag_ssim">
```

Comment:
i'm wondering if we could have a generic lazy_facet_component and just feed in the field name or something similar.